src/components/buttons/README.md

```diff
@@ -7,13 +7,13 @@ Buttons are created to help user to make actions, choices and to move around the
 ```jsx
 import {Button} from 'style-guide';
 
-<Button type="solid-blue" size="small">
+<Button type="solid-indigo" size="small">
   I am solid small button
 </Button>;
 ```
 
 ```HTML
-<div class="sg-button sg-button--solid-blue sg-button--small">
+<div class="sg-button sg-button--solid-indigo sg-button--small">
```

Comment:
Could we use `<button>` tag here, please? 🙏

Reply:
Done

Reply:
thanks!